### PR TITLE
Fix deadlock in SqlStageExecution.waitForNewExchangesOrBuffers

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/execution/SqlStageExecution.java [hunk 1]
```diff
@@ -51,6 +51,7 @@
 import io.airlift.stats.Distribution;
 import io.airlift.units.DataSize;
 import io.airlift.units.Duration;
+import org.eclipse.jetty.util.ConcurrentHashSet;
```
Comment (martint) [inline presto-main/src/main/java/com/facebook/presto/execution/SqlStageExecution.java]: Wrong import

**Comment by dain** [comment]
fixed

**Comment by dain** [comment]
Updated

**Comment by martint** [comment]
looks good

**Pull request merged**